app/models/concerns/key_pair_compute_resource.rb

```diff
@@ -23,8 +23,8 @@ def recreate
     setup_key_pair
   end
 
-  def delete_key_pair(kay_pair_name)
-    delete_key_from_resource(kay_pair_name)
+  def delete_key_pair(key_pair_name)
+    delete_key_from_resource(key_pair_name)
```

Comment:
It's not needed, I kept it because ComputeResource uses it but it could very well use `delete_key_from_resource(key_pair)`.  I noticed this change (removing the arguments from delete_key_from_resource) broke that call, so thanks for that 😄  Fixing..